src/pathfinding_service/database.py

```diff
@@ -86,3 +87,35 @@ def get_iou(
             return next(self.get_ious(sender, expiration_block, claimed))
         except StopIteration:
             return None
+
+    def upsert_channel_view(self, channel_view: ChannelView) -> None:
+        cv_dict = ChannelView.Schema(strict=True, exclude=['state']).dump(channel_view)[0]
+        self.conn.execute(
+            """
+            INSERT OR REPLACE INTO channel_view (
+                token_network_address, channel_id, participant1, participant2,
```

Comment:
I wonder if we should add the `token_network_registry` address here as well, just to be future safe. On the other hand this would request big changes anyways all over the client. so probably nvm.